Joplin_install_and_update.sh

```diff
@@ -202,16 +202,28 @@ if command -v lsb_release &> /dev/null; then
   DISTVER=$(lsb_release -is) && DISTVER=$DISTVER$(lsb_release -rs)
   DISTCODENAME=$(lsb_release -cs)
   DISTMAJOR=$(lsb_release -rs|cut -d. -f1)
+
   #-----------------------------------------------------
   # Check for "The SUID sandbox helper binary was found, but is not configured correctly" problem.
   # It is present in Debian 1X. A (temporary) patch will be applied at .desktop file
   # Linux Mint 4 Debbie is based on Debian 10 and requires the same param handling.
   #
-  # This also works around Ubuntu 23.10+'s restrictions on unprivileged user namespaces. Electron
+  # TODO: Remove: This is likely no longer an issue. See https://issues.chromium.org/issues/40462640.
+  BAD_HELPER_BINARY=false
+  if [[ $DISTVER =~ Debian1. || ( "$DISTVER" = "Linuxmint4" && "$DISTCODENAME" = "debbie" ) || ( "$DISTVER" = "CentOS" && "$DISTMAJOR" =~ 6|7 ) ]]; then
+    BAD_HELPER_BINARY=true
+  fi
```

Comment:
I've left this logic for now. The [suid helper binary seems to no longer be used/usable](https://issues.chromium.org/issues/40462640), but I think it makes more sense to remove this in a separate pull request, should this or the other pull request need to be reverted.